android/lib/AndroidPlatform.js

```diff
@@ -100,6 +102,16 @@ function(output, callback) {
         }
     });
 
+    // Checking env
+    var androidHome = "ANDROID_HOME";
+    msg = "Checking for " + androidHome + "... ";
+    if (!process.env[androidHome]) {
+        found = false;
+        output.error(msg + "empty!");
```

Comment:
1 - Please move the "good" case up, and save the negation.

2 - Would suggest adding something like
    output.error(androidHome + " needs to be set for builds to work");
    For a little better explanation.

Many thanks.